### ISSUE TITLE
does locale en-CH really exist?

### ISSUE THREAD

**Issue by AndyOGo** [axa-ch/aletheia#196]
Currently we always suffix the locale with `-CH`, but axa.ch only provides one of `de`, `en`, `fr`, `it`.

- [ ] `en-CH` isn't valid, i.d. officially registered (though it's allowed to extend locale information...) 🤔 
- [ ] we import localisation format only by language without respected `-CH` country code

**Comment by LucaMele** [comment]
nope, it does not exist.. they proposed it once to add English as a 5th National language thought... we look in the future!
